### PR TITLE
added folder set up, example data, data schema and updated gitignore

### PR DESCRIPTION
**Pull request by tonaramos**

### CODE REVIEW

File: aaaDataGenerator/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (rtsuzaki) [inline aaaDataGenerator/index.html:1]: Do you need this second HTML file? Can view mock data in the console

**Review by rtsuzaki — APPROVED** [review]

File: .babelrc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "presets": [
+        "env", "react"
+    ],
+    "plugins": ["transform-object-rest-spread"]
+}
```
Comment (clfhhc) [inline .babelrc:6]: follow airbnb style guide, lol

File: client/index.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>N-App-Listing-Description</title>
+    <meta name="description" content="description" >
```
Comment (clfhhc) [inline client/index.html:6]: better change the "description" in the "content"

File: package.json [hunk 4]
```diff
@@ -4,7 +4,8 @@
   "description": "N-appbnb description listing",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (clfhhc) [inline package.json:7]: can delete this line for now

File: sampleData.js [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+          "startDate": '2013-03-01',
+          "endDate": "2013-03-01", 
+          "nightsOfMinimumStay": 5
+          }],
+        "daysFromLastUpdate": 6
+      } 
+  }
+]
+
+let homeSchema = mongoose.Schema({
```
Comment (clfhhc) [inline sampleData.js:143]: why do you need to define your schema twice?

**Review by clfhhc — COMMENTED** [review]
The general setup looks good. I just have some suggestions for some details

**Pull request merged**